src/table/virtualized-table/Body.tsx

```diff
@@ -135,7 +149,7 @@ const Body = forwardRef<BodyRef, BodyProps>((props, ref) => {
       data-key="body"
       ref={gridRef}
       innerRef={innerForwardRef}
-      style={{ overflow: 'hidden' }}
+      style={listStyle}
```

Comment:
I do not think it is good to use the name listStyle for VariableSizeGrid. Also, VariableSizeGrid does not need boxSizing: 'border-box' as far as i see.

Reply:
listStyle for VariableSizeGrid was changed. Strange that you didn't see the change. But I didn't see your review comment 2 hours ago either. This is strange. By the way, where do you see `boxSizing: 'border-box'` property? I don't see that in Body.tsx